python/_spglib.c

```diff
@@ -34,8 +34,7 @@
 
 #include <Python.h>
 #include <assert.h>
-// TODO: Switch to newer numpy api
-#define NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION
+#define NPY_NO_DEPRECATED_API NPY_1_20_API_VERSION
```

Comment:
I chose 1.20 simply because that's the version on the latest RHEL9 (lower than Ubuntu 24.04). Any comments on this? I think adding a `build-system.requires` linked to this would be advisable?

Reply:
This seems fine to me.